### ISSUE TITLE
Google Home Hub compatibility?

### ISSUE THREAD

**Issue by scottcawley** [philippe44/AirConnect#274]
Hi, I’m having an issue with my Home Hub (one of the Google Devices with a screen - the smaller sized one) not showing up In the list of AirPlay devices. 

I’m running AirConnect in docker on my QNAP NAS, and it can find my Home mini speakers, and my TV Chromecasts - but the Home Hub doesn’t show up. I have set a speaker group for the Home Hub and one of my mini speakers - the group shows in the list, but when I AirPlay to it, sound only comes out of the mini, and the Home Hub is ignored.

I saw on another blog that someone seemed to have some success using AirConnect with the Home Hub Max (the newer, bigger screen device), so I am assuming what I am trying to do should be possible. But I am a bit lost why my Hub will not show. Could it be because this is running on my NAS, something wrong on the network, or is the device not compatible with AirConnect in the first place? Any pointers welcome! Thank you!

**Comment by philippe44** [comment]
It should be compatible, how is your HomeHub connected to your network? Same or different WiFi network, how is the connection between them? Have you disabled IGMP snooping? Is there something referring to multicast in your router?

**Comment by MrAct1on** [comment]
Sup,

Since I forgot to change my settings n keep getting these emails I might as well ask,

Is there a way to override or change the maximum amount of google nest speakers that u can connect to play music? 

I honestly don’t know shit about any of this but I managed to stumble my way into making it work. However, that was my only issue. If there’s no answer/solution, would u care for some free nest speakers?

Preesh
Mr. Action

**Comment by philippe44** [comment]
Not sure I understand what you want to do. You want to ignore some speakers so that they do not appear in AirPlay?

**Comment by PixPMusic** [comment]
Can confirm compatible with Home Hub. Are you still having issues?

**Comment by MrAct1on** [comment]
Naw. 

My problem was that I had too many google speakers. I tried grouping some speakers together like u can in google home, keeping them separate, then a mix of gen 1 vs gen 2 speaker combos. However, even with the variety not all of them would appear in airplay. 

That was the error that it gave when I finally got it to work. Like the instructions u gave, when u finally do it right a lot of code starts generating n what not. And it worked for about 10-12 speakers? Then the error came. I’ll look thru my screenshots to verify what it actually said but I’m pretty sure it was in the ballpark of a device limit. 

So I guess my problem was seeing if there was some way to change/increase that.
